index.html

```diff
@@ -45,5 +45,149 @@ <h1>I’m Roodjino Glad to see you!</h1>
             </div>
         </section>
 
+        <section class="work">
+
+            <div class="card-work">
+                <div class="card-img">
+                    <img src="images/portfolio1.png" alt="snapshoot">
+                </div>
+                    
+                <div class="card-text">
+                    <h2>Project Title</h2>
+                    <ul class="project-details">
+                        <li>Canopy</li>
+                        <span class="bullet"></span>
+                        <li>Black end Dev</li>
+                        <span class="bullet"></span>
+                        <li>2015</li>
+                    </ul>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="project-stack">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <a href="#"> <button type="button">See Project</button> </a>
```

Comment:
- Please, let's add the interaction of the buttons to indicate the `active`, `hover`, and `pressed` states to improve the user experience. kindly see it in the Figma design template.

Reply:
Thank you for your feedback! I will do it

Reply:
> * Please, let's add the interaction of the buttons to indicate the `active`, `hover`, and `pressed` states to improve the user experience. kindly see it in the Figma design template.

It's done @zieeco !